examples/src/main/java/org/bitcoinj/examples/RefreshWallet.java

```diff
@@ -27,6 +27,15 @@
 
 /**
  * RefreshWallet loads a wallet, then processes the block chain to update the transaction pools within it.
+ * 
+ * To get a test wallet run the following command against the tools subproject: <br>
```

Comment:
I think rather than providing the exact command line it would be enough to just refer to wallet-tool.